### PR TITLE
removed famfamfam icon set

### LINKED ISSUE #4314: Remove famfam icons
We moved from glyphicons to font-awesome, but still have the famfam icons in our project:

https://github.com/sonata-project/SonataAdminBundle/tree/master/Resources/public/famfamfam

Based on comment https://github.com/sonata-project/SonataAdminBundle/issues/3291#issuecomment-278282385, the should be removed.

Should this be done against `master`?

### PR DESCRIPTION
**Pull request by OskarStark**
Closes #4314 

## Changelog

```markdown
### Removed
- famfamfam icon set
```

### CODE REVIEW

**Comment by OskarStark** [comment]
😄 
screenshot 2017-02-08 11 05 36

**Comment by jvasseur** [comment]
This should probably go to master, it is kind of a BC break if someone was relying on sonata have those icons available.

**Comment by OskarStark** [comment]
> This should probably go to master, it is kind of a BC break if someone was relying on sonata have those icons available.

Yeah, this is my thought, too

**Comment by greg0ire** [comment]
Let's play it safe then.

**Comment by OskarStark** [comment]
Updated and now against `master`

**Comment by OskarStark** [comment]
> Please add an UPGRADE note

Done, are you fine with it?

File: UPGRADE-4.0.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+## Removed `famfamfam` icon set
+
+If you still need it, please add it by your own!
```
Comment (greg0ire) [inline UPGRADE-4.0.md]: "please set it up it on your own" or, "please set it up yourself"

Comment (OskarStark) [inline UPGRADE-4.0.md]: done

**Comment by OskarStark** [comment]
ping @greg0ire

File: UPGRADE-4.0.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+## Removed `famfamfam` icon set
+
+If you still need it, please set it up by your own!
```
Comment (greg0ire) [inline UPGRADE-4.0.md]: "on your own"

**Comment by core23** [comment]
Do we still use this in any template?

**Comment by OskarStark** [comment]
> Do we still use this in any template?

TO be honest, not sure how to test this...

**Comment by core23** [comment]
Don't know the famfamfam icons set, but there should be some references like `class="fa fa-list"` or anything else.

**Comment by OskarStark** [comment]
I don't think so, there is no css which belongs to them

**Comment by OskarStark** [comment]
Imo we should merge this, as it is in `master`and not released yet, so if we found some problems afterwards we could patch them.

Or a bash guru can get all file names of the icons and grep through all sonata bundles...

**Comment by mvhirsch** [comment]
Bash guru here, just did a short grep of "famfamfam" in one of my projects which uses nearly all sonata-bundles. Found matches:

```
$> grep -R famfamfam sonata-project/

sonata-project/admin-bundle/Resources/meta/LICENSE:http://www.famfamfam.com/lab/icons/silk/
sonata-project/page-bundle/Resources/views/PageAdmin/select_site.html.twig:                        <img src="{{ asset('bundles/sonataadmin/famfamfam/accept.png') }}">
```

The first match can be ignored (this MR removes it anyway). The second one is in PageBundle, the usage of the image should be removed too. Of course, this needs to be adressed on all bundles (nearly all != all) ... :-)

**Comment by OskarStark** [comment]
Thank you @mvhirsch 

This PR can be merged after: https://github.com/sonata-project/SonataPageBundle/pull/787

**Comment by OskarStark** [comment]
@greg0ire can you please push the button ;-)

**Comment by greg0ire** [comment]
Sure!

**Pull request merged**